docs/troubleshooting.md

```diff
@@ -0,0 +1,13 @@
+# Troubleshooting
+
+## "Undetermined" State Interactive Apps
+
+When launching an interactive app, you may see an information card like this one:
+
+![Screenshot of an interactive app information card with a yellow warning header that reads "Code Server \( \[1\) Undetermined"](images/troubleshooting_undetermined_1.png)
+
+If this happens, it is usually because of a problem in your `.bashrc` file. If you use the [dashboard terminal app](terminal.md) to connect to a login node, you should see the full error causing the problem. The error message should point to a solution to apply to your `.bashrc` file, but if you have trouble finding a solution, reach out to [ithelp@harvard.edu](mailto:ithelp@harvard.edu?subject=HUIT Open OnDemand) for further assistance.
```

Comment:
Should we add a note for the failing `spack load <module>`. Something like "Note: loading spack modules directly in the bashrc could also cause failing sessions, some work arounds are ...."

Reply:
We should include something about that, but the real solution is to use a spack environment, which I think requires either write access to the global spack directory or a downstream spack installation. Since we don't have a good fix, it kind of does boil down to "load a shell, see what's wrong, and fix it". Once we have a pattern for using environments, we'll want to use it here.